client/my-sites/people/team-invites/index.tsx

```diff
@@ -16,6 +16,8 @@ function TeamInvites() {
 	const siteId = site?.ID as number;
 	const pendingInvites = useSelector( ( state ) => getPendingInvitesForSite( state, siteId ) );
 
+	useGetInvitesQuery( siteId );
```

Comment:
This could be just me, but I find this bare use of a hook here a bit strange especially when it's named `useGetInvitesQuery`. Is it possible to perhaps return `requestSiteInvites` and move the dispatch call here?

Reply:
I didn't want to introduce handling window focus events if there is a generic way of it by relying on `useQuery`, so fetching team members and pending invites in the same way.

Also, there is a convention to name a query hook starting with `use` and ending with `query` at the end. Maybe I can rename it to `useInvitesQuery`, it's a bit better. :)